runbotics/runbotics-scheduler/src/database/process-instance-event/process-instance-event.service.ts

```diff
@@ -16,7 +16,18 @@ export class ProcessInstanceEventService {
         return event;
     }
 
+    async update(event: IProcessInstanceEvent) {
```

Comment:
imo async/await keywords are redundant. If you just return the repository method, the return type of this fn will be the same

Reply:
done